aiidalab_widgets_base/computers.py

```diff
@@ -450,161 +455,145 @@ def __private_key(self):
-                                             description="Mpirun command:",
-                                             layout=ipw.Layout(width="500px"),
-                                             style=STYLE)
-        self._computer_ncpus = ipw.IntText(value=12,
-                                           step=1,
-                                           description='Number of CPU(s) per node:',
-                                           layout=ipw.Layout(width="270px"),
-                                           style=STYLE)
-        self._transport_type = ipw.Dropdown(value='ssh',
-                                            options=Transport.get_valid_transports(),
-                                            description="Transport type:",
+        # List of widgets to be displayed.
+        inp_computer_name = ipw.Text(value='',
+                                     placeholder='Will only be used within AiiDA',
+                                     description="AiiDA computer name:",
+                                     layout=ipw.Layout(width="500px"),
+                                     style=STYLE)
+        link((inp_computer_name, 'value'), (self, 'label'))
+
+        # Hostname.
+        inp_computer_hostname = ipw.Text(description="Hostname:", layout=ipw.Layout(width="500px"), style=STYLE)
+        link((inp_computer_hostname, 'value'), (self, 'hostname'))
+
+        # Computer description.
+        inp_computer_description = ipw.Text(value='',
+                                            placeholder='No description (yet)',
+                                            description="Computer description:",
+                                            layout=ipw.Layout(width="500px"),
                                             style=STYLE)
-        self._scheduler = ipw.Dropdown(value='slurm',
-                                       options=Scheduler.get_valid_schedulers(),
-                                       description="Scheduler:",
-                                       style=STYLE)
+        link((inp_computer_description, 'value'), (self, 'description'))
+
+        # Directory where to run the simulations.
+        inp_computer_workdir = ipw.Text(value='/scratch/{username}/aiida_run',
+                                        description="Workdir:",
+                                        layout=ipw.Layout(width="500px"),
+                                        style=STYLE)
+        link((inp_computer_workdir, 'value'), (self, 'work_dir'))
+
+        # Mpirun command.
+        inp_mpirun_cmd = ipw.Text(value='mpirun -n {tot_num_mpiprocs}',
+                                  description="Mpirun command:",
+                                  layout=ipw.Layout(width="500px"),
+                                  style=STYLE)
+        link((inp_mpirun_cmd, 'value'), (self, 'mpirun_command'))
+
+        # Number of CPUs per node.
+        inp_computer_ncpus = ipw.IntText(value=12,
```

Comment:
Why is 12 the default?

Reply:
Any number is fine. I can also set it to 1. I don't have any preferences here.